### PR TITLE
fix(PE-5222): update max length to 61

### PR DESCRIPTION
**Pull request by atticusofsparta**

### CODE REVIEW

**Comment by dtfiedler** [comment]
add a unit test and surprised to not see an AJV change

File: src/actions/write/setRecord.test.ts [hunk 1]
```diff
@@ -25,18 +25,27 @@ import { baselineAntState } from '../../../tests/utils/constants';
 import { setRecord } from './setRecord';
 
 describe('setRecord', () => {
+  let state = { ...baselineAntState };
+
+  beforeEach(() => {
+    state = { ...baselineAntState };
+  });
```
Comment (dtfiedler) [inline src/actions/write/setRecord.test.ts:32]:
```suggestion
```

File: tsconfig.json [hunk 2]
```diff
@@ -12,7 +28,7 @@
     "allowJs": true,
     "baseUrl": ".",
     "types": ["jest"],
-    "lib": ["ESNext"]
+    "lib": ["ESNext", "DOM"]
```
Comment (dtfiedler) [inline tsconfig.json:31]: 🤔

Comment (atticusofsparta) [reply tsconfig.json:31]: for console

File: src/actions/write/setRecord.test.ts (hunk 1 above)
Comment (atticusofsparta) [reply src/actions/write/setRecord.test.ts]: removed - thought this was needed when trying to debug a test. Misunderstood how padEnd worked 😓

File: tsconfig.json (hunk 2 above)
Comment (dtfiedler) [reply tsconfig.json:31]: Not sure I follow - why do we need Dom types available?

Comment (atticusofsparta) [reply tsconfig.json:31]: Sorry, to be more specific TS was complaining about use of console and suggested adding the dom type. Node type would probably also work.

Comment (dtfiedler) [reply tsconfig.json:31]: where was it complaining?

Comment (atticusofsparta) [reply tsconfig.json:31]: in a test - i took the console out. I guess we dont actually need it, just needed it at the time for debugging

Comment (dtfiedler) [inline tsconfig.json:31]:
```suggestion
    "lib": ["ESNext"]
```

**Pull request merged**